mobile/README.md

```diff
@@ -38,6 +61,26 @@ To checkout the latest tagged release of lnd, run
 ⛰  git checkout $(git describe --tags --abbrev=0)
 ```
 
+Or, alternatively, you can clone the project and checkout to any branch:
+
+```shell
+⛰  git clone https://github.com/lightningnetwork/lnd
```

Comment:
If the user already is in `$GOPATH/src/github.com/lightningnetwork/lnd` then this will create the directory `$GOPATH/src/github.com/lightningnetwork/lnd/lnd`. So we should either add a `cd ..` first or add a remark that the user needs to be in the `$GOPATH/src/github.com/lightningnetwork/lnd` directory and then clone into the current directory directly with `git clone https://github.com/lightningnetwork/lnd .` (note the dot in the end).